### DISCUSSION TITLE
iCub sholder yaw joint fault !

### DISCUSSION THREAD

**Discussion by ivan9a** [robotology/community#515] [Q&A] [answered]
Hi, 

We have major malfunctions with yaw joint on the left arm, please look at the pictures down below:
![IMG_20210629_162719](https://user-images.githubusercontent.com/83449933/123931944-453c5800-d991-11eb-81cd-ff7079f01ed5.jpg)
![IMG_20210629_162813](https://user-images.githubusercontent.com/83449933/123931965-49687580-d991-11eb-9234-5aba1231b59b.jpg)
![IMG_20210629_163020](https://user-images.githubusercontent.com/83449933/123931969-4a99a280-d991-11eb-90a1-f76f9255f7c8.jpg)
![IMG_20210629_163032](https://user-images.githubusercontent.com/83449933/123931975-4bcacf80-d991-11eb-8005-3294ecd26c46.jpg)
![IMG_20210629_163039](https://user-images.githubusercontent.com/83449933/123931987-4cfbfc80-d991-11eb-96f5-3d20d53db45d.jpg)
![IMG_20210629_163046](https://user-images.githubusercontent.com/83449933/123931992-4d949300-d991-11eb-9ccf-ac7d3d784344.jpg)

We have only run your redball demo, during initialization one steel cable just broke and everything suddenly stopped. Do you have any instructions on how to replace steel cables ? Can you provide new or replacing parts for that fault joint ?

Thank you.

Best regards,
Ivan Zupancic

**Comment by pattacini** [comment, marked as answer]
Hi @ivan9a 

Unfortunately, this is not the right place where filing tickets for mechanical support.
Could you please open up an issue at https://github.com/robotology/icub-tech-support/issues instead?

This is documented in our [General Rules for Posting][1].

[1]: https://github.com/robotology/community/blob/master/.github/SUPPORT.md

**Reply by ivan9a** [reply]
Hi,

Ok it is my mistake, i will post at **/icub-tech-support/issues** 

Thank you,
Ivan Zupancic

**Reply by pattacini** [reply]
Thanks a lot @ivan9a 👍🏻
